lib/ts.jte

```diff
@@ -49,8 +49,8 @@ JARPATH=.
 # Adjust these as necessary for Windows or UNIX
 # (Must use "${JARPATH}" syntax so ant will expand the nested properties)
 
-##LOCAL_CLASSES=${JARPATH}\\jakarta.mail.jar:${JARPATH}\\jakarta.activation.jar
-LOCAL_CLASSES=${JARPATH}/jakarta.mail.jar:${JARPATH}/jakarta.activation.jar
+##LOCAL_CLASSES=${JARPATH}/jakarta.mail-api.jar:${JARPATH}/jakarta.activation-api.jar:${JARPATH}/angus-mail.jar:${JARPATH}/angus-activation.jar
+LOCAL_CLASSES=${JARPATH}/jakarta.mail-api.jar:${JARPATH}/jakarta.activation-api.jar:${JARPATH}/angus-mail.jar:${JARPATH}/angus-activation.jar
```

Comment:
we want to have two variables - one for API jar files (*-api.jar), the other for implementation jar files (ie angus-*.jar)
we want the execution command to look like:
`$JAVA_HOME/bin/java --add-modules jakarta.mail --module-path $API_JARS -verify $testExecuteClass $testExecuteArgs $testDebug`

(APIs on the module path, the rest remains on the classpath)

see the change in jaf-tck: https://github.com/eclipse-ee4j/jaf-tck/pull/50/commits/86994a4274d1e4f4143961c6e93bbfb252c45ae4

Reply:
done.